rootfs/api/models/release.py

```diff
@@ -24,6 +24,7 @@ class Release(UuidAuditedModel):
     version = models.PositiveIntegerField()
     summary = models.TextField(blank=True, null=True)
     failed = models.BooleanField(default=False)
+    deployed = models.BooleanField(default=True)
```

Comment:
I'm in favour of this feature overall, however this really irks me. We should not be maintaining "cluster state" in the database. Rather, I feel like any check to see if the release is deployed should be a call out to kubernetes to see if the deployment is present or not.

What do you think?